Magento2/Tests/Legacy/ModuleXMLUnitTest.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Magento_TestModule" active="true" version="1" />
```

Comment:
I'd add more test cases
```suggestion
    <module name="Magento_TestModule" active="true" version="1" />
    <module 
        name="Magento_TestModule" 
        active="true" 
        version="1" 
    />
```

Reply:
I thought about it, but do `module.xml` contain more than a single `module` tag? If you consider it worthwile, I can add this case as well.

Reply:
It would be good to test both structures

Reply:
@svera, it should have two on more module nodes. Please extract it to a separate xml file